examples/data_ops/1151_digits.py

```diff
@@ -0,0 +1,165 @@
+"""
+Handwritten digit classification with skrub
+============================================
+
+This example demonstrates how to use skrub's Data Ops to build a
```

Comment:
maybe this sentence could say why this example is here / what it adds wrt to the other examples? e.g. illustrate dataops can be useful on non-dataframish data or something like that